src/aiq_agent/agents/deep_researcher/sandbox/README.md

```
@@ -0,0 +1,289 @@
+
+- `AIQ_MAX_SANDBOXES_PER_PRINCIPAL` / `AIQ_MAX_SANDBOXES_GLOBAL` (default-off): submit-path
+  concurrency/cost caps for sandbox-enabled jobs.
+- `AIQ_OPENSHELL_ADAPTER_FILE_TRANSFER` (default-off): route OpenShell file transfer through
+  the official adapter instead of the env-free shim (see OpenShell gotcha above).
+- Artifact retention reuses the job-expiry periodic cleanup (`expiry_seconds`).
+
+## Testing
+
+```bash
+pytest tests/aiq_agent/agents/deep_researcher/sandbox/ -q
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)